MVP/defaults-build.yaml

```diff
@@ -1,6 +1,23 @@
 - defaults:
     name: defaults-build
     node: 'jslave-platform-rhel7'
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag
```

Comment:
Hmm.. I believe the folk who tries to trigger a job should already have pretty good understanding about those vars as they are being used in job definition for tests.
Otherwise they are probably not required, thus can be left blank.